account_move_report/views/account_move_report.xml

```diff
@@ -63,10 +65,12 @@
                                             <span t-field="l.credit"/>
                                         </td>
                                         <td><span t-field="l.analytic_account_id"/></td>
-                                        <td class="text-right">
-                                            <span t-field="l.amount_currency"/>
-                                        </td>
-                                        <td><span t-field="l.currency_id.name"/></td>
+                                        <t t-if="l.amount_currency">
```

Comment:
@luistorresm    
What if `amount_currency` is zero?

Reply:
Could you send me an example, please?

Regards

Reply:
@luistorresm   
Yes. Moves corresponding to exchange difference entries could have 0 in `amount_currency`, but could have set `currency_id`.

AFAIK, the right way to know if an amount is on a different currency, is to check if `currency_id is set`